### ISSUE TITLE
test[4724] failed - expected:<120> but was:<0>

### ISSUE THREAD

**Issue by pointhi** [graalvm/sulong#423]
#### System Informations

x86_64 - [Antergos](https://antergos.com) ([archlinux](https://www.archlinux.org) based distribution)

I'm using gcc-4.7 instead of gcc-4.6, because of some building problems.

#### JUnit Error

```java
test[4724](com.oracle.truffle.llvm.test.TestGCCSuite)
java.lang.AssertionError: /tmp/compiled-llvm-tests/.projects.com.oracle.truffle.llvm.test.suites.gcc.gcc-5.2.0.gcc.testsuite.gcc.dg.autopar.pr39500-2._main.temp.ll expected:<120> but was:<0>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at com.oracle.truffle.llvm.test.TestGCCSuite.test(TestGCCSuite.java:87)
	at sun.reflect.GeneratedMethodAccessor73.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runners.Suite.runChild(Suite.java:127)
	at org.junit.runners.Suite.runChild(Suite.java:26)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runners.Suite.runChild(Suite.java:127)
	at org.junit.runners.Suite.runChild(Suite.java:26)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at com.oracle.mxtool.junit.MxJUnitWrapper.main(MxJUnitWrapper.java:172)
```

#### /tmp/compiled-llvm-tests/.projects.com.oracle.truffle.llvm.test.suites.gcc.gcc-5.2.0.gcc.testsuite.gcc.dg.autopar.pr39500-2._main.temp.ll

```llvm
; ModuleID = '/home/thomas/Dokumente/JKU/sulong-dev/sulong/projects/com.oracle.truffle.llvm.test/suites/gcc/gcc-5.2.0/gcc/testsuite/gcc.dg/autopar/pr39500-2.c'
target datalayout = "e-p:64:64:64-i1:8:8-i8:8:8-i16:16:16-i32:32:32-i64:64:64-f32:32:32-f64:64:64-v64:64:64-v128:128:128-a0:0:64-s0:64:64-f80:128:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"

define i32 @main() nounwind uwtable {
  %1 = alloca i32, align 4
  %i = alloca i32, align 4
  %x = alloca [1000 x i32], align 16
  store i32 0, i32* %1
  store i32 0, i32* %i, align 4
  br label %2

; <label>:2                                       ; preds = %14, %0
  %3 = load i32* %i, align 4
  %4 = icmp slt i32 %3, 101
  br i1 %4, label %5, label %17

; <label>:5                                       ; preds = %2
  %6 = load i32* %i, align 4
  %7 = add nsw i32 %6, 100
  %8 = sext i32 %7 to i64
  %9 = getelementptr inbounds [1000 x i32]* %x, i32 0, i64 %8
  %10 = load i32* %9, align 4
  %11 = load i32* %i, align 4
  %12 = sext i32 %11 to i64
  %13 = getelementptr inbounds [1000 x i32]* %x, i32 0, i64 %12
  store i32 %10, i32* %13, align 4
  br label %14

; <label>:14                                      ; preds = %5
  %15 = load i32* %i, align 4
  %16 = add nsw i32 %15, 1
  store i32 %16, i32* %i, align 4
  br label %2

; <label>:17                                      ; preds = %2
  %18 = getelementptr inbounds [1000 x i32]* %x, i32 0, i64 12
  %19 = load i32* %18, align 4
  ret i32 %19
}
```

**Comment by mrigger** [comment]
Thanks for the issue! It seems that this test case is not meant to be executed since its result depends on an uninitialized stack array. I will exclude it from the test suite.

**Comment by mrigger** [comment]
Fixed with #424.
